.github/workflows/ci-build.yml

```diff
@@ -46,21 +47,22 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python }}
+      - name: Install poetry
+        run: |
+          pipx install poetry
       - name: Build a binary wheel
         run: |
-          python -m pip install numpy --compile --pre
-          python -m pip install --user -U pip wheel setuptools
-          python setup.py bdist_wheel
+          poetry build
       - name: Install LedFx
         run: |
-          pip install --pre --find-links=dist .
+          poetry install
 
   build-ledfx-osx:
     name: Build LedFx (OS X)
     runs-on: macos-latest
     strategy:
       matrix:
-        python: [3.8,3.9,3.10.x]
+        python: [3.8,3.9,3.10.x,3.11.x,3.12.x]
```

Comment:
Recommend drop 3.8

Reply:
We'll have to tinker with the CI so can do that later.

Given our wheel is pure python we may not need to do matrix builds for every OS since the build should in theory always work - if LedFx functions may be another question...